elide-async/src/main/java/com/yahoo/elide/async/service/AsyncQueryDAO.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.async.service;
+
+import java.util.UUID;
+
+import com.yahoo.elide.Elide;
+import com.yahoo.elide.async.models.AsyncQuery;
+import com.yahoo.elide.async.models.AsyncQueryResult;
+import com.yahoo.elide.core.DataStore;
+
+/**
+ * Utility interface which uses the elide datastore to modify, update and create
+ * AsyncQuery and AsyncQueryResult Objects
+ */
+public interface AsyncQueryDAO {
+
+    /**
+     * Set elide object
+     * @param elide Elide Object.
+     */
+    public void setElide(Elide elide);
```

Comment:
This seems like a constructor argument rather than an interface contract.  I would remove.